devicon.json

```
@@ -7444,6 +7444,27 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "processwire",
```

Add empty altnames (all root level attributes need to be provided)
```suggestion
        "name": "processwire",
        "altnames": [],
```